projects/github-actions/repo-gardening/src/tasks/notify-kitkat/index.js

```diff
@@ -137,6 +137,12 @@ async function notifyKitKat( payload, octokit ) {
 		return;
 	}
 
+	// Only proceed if the issue is stil open.
+	if ( 'open' !== state ) {
+		debug( `notify-kitkat: Issue #${ number } is already closed. Aborting.` );
```

Comment:
Might be more useful to note the actual state in the debug log.
```suggestion
		debug( `notify-kitkat: Issue #${ number } is state '${ state }'. Aborting.` );
```